### ISSUE TITLE
Downloaded SVG image is corrupted when accessibility.js script included

### ISSUE THREAD

**Issue by ressbill** [highcharts/highcharts#16836]
#### Expected behaviour
Downloaded svg image opens successfully

#### Actual behaviour
When accessibility.js script is included then generated svg image contains an error and cannot be opened.
**Error msg:** This page contains the following errors:
error on line 1 at column 284: Extra content at the end of the document

#### Live demo with steps to reproduce
Reproduced on the official site:
https://www.highcharts.com/demo/bar-basic

Jsfiddle:
https://jsfiddle.net/9028bjxh/
#### Product version
latest v9.3.2 versions

**Comment by raf18seb** [comment]
Closing as a duplicate of https://github.com/highcharts/highcharts/issues/16589
